pubsub/kafka/metadata.yaml

```diff
@@ -293,7 +293,7 @@ metadata:
         Enables caching for schemas.
       example: '"true"'
       default: '"true"'
-    - name: SchemaLatestVersionCacheTTL
+    - name: schemaLatestVersionCacheTTL
```

Comment:
Can you please fix this in https://github.com/dapr/components-contrib/blob/85722a4e8817f18dbb9406fa99c86c47712380d1/bindings/kafka/metadata.yaml#L305 too?

Reply:
sorry I thought I did but didn't save the file! doh!